File: day010/day010.cs
using System;

class Day010 {
  static void Main() {
    int x;
    
    Console.Write("Bir sayı giriniz: ");
    x = Convert.ToInt32(Console.ReadLine());
    
    if (x < 0)
        Console.WriteLine("Girdiğiniz sayı negatiftir.");
    
	  else if (x > 0)
        Console.WriteLine("Girdiğiniz sayı pozitiftir.");
    
	  else
        Console.WriteLine("Girdiğiniz sayı nötrdür.");
  }
}

File: day010/day010.cpp
#include <iostream>

using namespace std;

int main()
{
    int x;
    
	  cout<<"Bir sayı giriniz: ";
      cin >> x;
    
    if (x != 0 && x < 0)
      cout << "Girdiğiniz sayı negatiftir.";

    else if (x != 0 && x > 0)
      cout << "Girdiğiniz sayı pozitiftir.";

    else
      cout << "Girdiğiniz sayı nötrdür";
    
    return 0;
}
